types/src/state_store/state_slot.rs

```diff
@@ -10,7 +10,8 @@ use crate::{
     transaction::Version,
 };
 use aptos_crypto::{hash::CryptoHash, HashValue};
-use derivative::Derivative;
+#[cfg(any(test, feature = "fuzzing"))]
+use proptest::prelude::*;
 use StateSlot::*;
```

Comment:
just curious, why is this use needed here (in the same file as StateSlot itself)?